packages/rollup-plugin/src/index.ts

```diff
@@ -78,6 +78,21 @@ export function stylableRollupPlugin({
         async buildStart() {
             extracted = extracted || new Map();
             emittedAssets = emittedAssets || new Map();
+
+            const configuration = resolveStcConfig(
+                projectRoot,
+                typeof stcConfig === 'string' ? stcConfig : undefined,
+                fs
+            );
+
+            if (stcConfig && !configuration) {
+                throw new Error(
```

Comment:
does it ever reach this point? wouldn't the internal `require.resolve` throw before reaching this point?

Reply:
I believe you're right, removing this code.